domino/airflow/__init__.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from domino.airflow._operator import DominoOperator
+except SyntaxError:
```

Comment:
If we don't do this python throws a compile time exception (due to `SyntaxError` because typing is a language feature) which means the entire module can't be loaded.